### ISSUE TITLE
Bug when zooming timeline too fast

### ISSUE THREAD

**Issue by nikakhov** [AnalyticalGraphicsInc/cesium#4425]
Steps to reproduce:

1.  Load https://cesiumjs.org/Cesium/Apps/HelloWorld.html
2.  Put cursor over timeline and give it one fast scroll (two fingers on touchpad)
3.  Try zooming a few more times and it should not zoom anymore

I think the bug is in the code below.  It goes away if you make `timelineWheelDelta ` constnat, but this decouples zooming speed from scrolling speed.

```
function createMouseWheelCallback(timeline) {
        return function(e) {
            var dy = e.wheelDeltaY || e.wheelDelta || (-e.detail);
            timelineWheelDelta = Math.max(Math.min(Math.abs(dy), timelineWheelDelta), 1);
            dy /= timelineWheelDelta;
            timeline.zoomFrom(Math.pow(1.05, -dy));
        };
    }
```

**Comment by hpinkos** [comment]
Thanks @nikakhov, I was able to reproduce this

If you don't have a touchpad, keep scrolling out until the timeline goes from like year 100 - 1000.  Eventually the scroll will stop working.  Additionally, if you scrub to the beginning of the timeline, you get an invalid ISO8601 date  ( would be these two things are related)

**Comment by nikakhov** [comment]
I think the problem is that after you scroll too fast, timelineWheelDelta will be equal to 1 indefinitely.

**Comment by hpinkos** [comment]
Hmm okay, that might be a different issue than I saw.  Thanks for clarifying!  We'll look into solving both problems.

**Comment by hpinkos** [comment]
Actually @nikakhov, if you think you have a good fix for the issue you saw, would you be interested in submitting a pull request?  Check out our contributing guide for information about steps to take to submit a PR: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md

**Comment by nikakhov** [comment]
If I eventually come up with a solution, then I would be happy to contribute.  At this point, I only have ideas on what the cause might be.

**Comment by hpinkos** [comment]
Okay thanks!  If you have any questions let me know, I'd be happy to help

**Comment by erikmaarten** [comment]
@nikakhov Do you still see the same issue if you try it here: http://cesium-dev.s3-website-us-east-1.amazonaws.com/cesium/timeline/Apps/HelloWorld.html?

**Comment by nikakhov** [comment]
Yes, if you scroll fast a few times back and forth, it will get stuck.

**Comment by erikmaarten** [comment]
@nikakhov Thanks for reporting back. Would you like to submit a pull request with the fix you found? ([Step-by-step guide here](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md), and in case you have any questions we're happy to help.)

**Comment by nikakhov** [comment]
I'm looking into this.

**Comment by erikmaarten** [comment]
@nikakhov How is this going? Have you stumbled on any problem that we can help with?

**Comment by nikakhov** [comment]
I created a pull request:

https://github.com/AnalyticalGraphicsInc/cesium/pull/4693

When I tested it, under the same conditions, it does not get stuck anymore, but you do get "undefined" in tick labels.
